### ISSUE TITLE
3ds twilight menu++ can't play pokemon heart gold vietnamese game 

### ISSUE THREAD

**Issue by Pig0983** [DS-Homebrew/TWiLightMenu#2272]
My 3ds run twilight menu++ latest version but i can't run pokemon heart gold vietnamese game,please fix it bro.[Aowvn.org](https://www.aowvn.org/2023/01/nds-game-pokemon-soulsilver-va.html?m=1) here my website where i download

**Comment by RocketRobz** [comment]
What issue are you having with the game, exactly?

**Comment by Pig0983** [comment]
When i go to the game the white screen appear bro

**Comment by RocketRobz** [comment]
> When i go to the game the white screen appear bro

When you launched the ROM for the first time, did the nds-bootstrap logo appear?
Also, have you tried any other ROMs?

**Comment by Pig0983** [comment]
Yes the ndd-bootstrap logo appear but after 30s-1minute later the white screen appear, only pokemon heart gold and storm silver vietnamese rom bro
